plugin/db/spanner/spanner.go

```diff
@@ -110,6 +142,14 @@ func (*Driver) Query(_ context.Context, _ string, _ *db.QueryContext) ([]interfa
 	panic("not implemented")
 }
 
+func (d *Driver) composeDSN() string {
+	return d.composeDSNWithDatabase(d.config.Database)
+}
+
+func (d *Driver) composeDSNWithDatabase(database string) string {
```

Comment:
avoid using word compose

getDSN(database string)

Reply:
changed to func getDSN(host, database string) string